lib/src/stream_channel_transformer.dart

```diff
@@ -23,8 +23,8 @@ import 'transformer/typed.dart';
 /// the stream to close before it emits any more events. This guarantee is
 /// invalidated when an asynchronous gap is added between the original stream's
 /// event dispatch and the returned stream's, for example by transforming it
-/// with a [StreamTransformer]. The guarantee can be easily preserved using [new
-/// StreamChannel.withCloseGuarantee].
+/// with a [StreamTransformer]. The guarantee can be easily preserved using
+/// [new StreamChannel.withCloseGuarantee].
```

Comment:
This change isn't necessary; the existing format is already parsed propely. Same for similar changes below.